tests/conftest.py

```diff
@@ -0,0 +1,29 @@
+"""Conftest File is for all the unit test files that test file logging."""
```

Comment:
nit: this is slightly not true.  This file is for all unit tests in this folder and any subfolders, so this is just a collection of fixtures and utilities that might be useful for multiple unit tests.